gpu_bdb/benchmark_runner/slurm/run_bench.sh

```diff
@@ -0,0 +1,55 @@
+set -e pipefail
```

Comment:
This is a combination of the `scheduler-client` and `spawn-workers` scripts which can be removed.